cappy.py

```diff
@@ -123,11 +137,12 @@ def handle_inbound_request(self, message):
             request_id = message['id']
             args = message['args']
         except KeyError as e:
+            print(e)
```

Comment:
There's an `e` there so I thought we might as well do something with it, even if just printing it, otherwise it's just an unused variable.